c/xudt_rce.c

```diff
@@ -37,9 +37,13 @@ int ckb_exit(signed char);
 #define MAX_CODE_SIZE (1024 * 1024)
 #define FLAGS_SIZE 4
 #define MAX_LOCK_SCRIPT_HASH_COUNT 2048
-#define OWNER_MODE_MASK 0xC0000000
+
 #define OWNER_MODE_INPUT_TYPE_MASK 0x80000000
 #define OWNER_MODE_OUTPUT_TYPE_MASK 0x40000000
+#define OWNER_MODE_INPUT_LOCK_NOT_MASK 0x20000000
+#define OWNER_MODE_MASK                                       \
+  (OWNER_MODE_INPUT_TYPE_MASK + OWNER_MODE_OUTPUT_TYPE_MASK + \
```

Comment:
`+` probably works here as well, but shall we `|` those masks together? Using plus here looks quite weird